packages/devtools_app/web/index.html

```diff
@@ -26,6 +26,16 @@
         if (!supportsES6Classes()) {
             window.location.href = '/unsupported-browser.html';
         }
+
+        // Handle legacy dart:html app URLs by moving query strings over to
```

Comment:
are legacy dart:html app urls from launch attempts from older vs code plugin versions?

Reply:
Oops, I didn't keep the comment up-to-date. Right now, all VS Code extensions are going to launch with old URLs, as the VS Code bug mentioned above prevents us from building URLs using the fragment (VS Code encodes them badly - a long-standing bug that can't be fixed because it's a breaking change, so needs additional work to fix).

Reply:
Tweaked the comment and added a ref to the Code issue.